src/features/page_view_event/instrument/index.js

```diff
@@ -28,6 +31,29 @@ export class Instrument extends InstrumentBase {
     }
     // Else, inference: inside worker or some other env where these events are irrelevant. They'll get filled in with 0s in RUM call.
 
+    // For now we are just capturing supportability metrics on `console` usage to assess log forwarding feature.
+    const consoleEE = wrapConsole(this.ee)
```

Comment:
Since this is directly tied to sending a `sm` at the moment, we should put the wrapping and handling code in the metrics instrument file instead of the PVE instrument file to save space in the lite build.

Reply:
This has been moved.